playbooks/roles/edxapp/files/update_theme.sh

```diff
@@ -17,7 +17,7 @@ if [ "$1" == "true" ] || [ "$1" == "True" ]; then
 	sudo git clone https://github.com/microsoft/edx-theme.git themes -b oxa/master.fic
```

Comment:
I agree with you Elton. hardcoded is not good. But this is our production theming and only we will use it. Let's go like that and I promise I will fix that later. It is harmless for now.